python/lsst/analysis/tools/actions/plot/histPlot.py

```diff
@@ -216,56 +270,118 @@ def _assignColors(self):
             ],
         )
         if self.cmap in custom_cmaps.keys():
-            all_cols = custom_cmaps[self.cmap]
+            all_colors = custom_cmaps[self.cmap]
         else:
             try:
-                all_cols = getattr(plt.cm, self.cmap).copy().colors
+                all_colors = getattr(plt.cm, self.cmap).copy().colors
             except AttributeError:
                 raise ValueError(f"Unrecognized color map: {self.cmap}")
 
         counter = 0
-        cols = defaultdict(list)
+        colors = defaultdict(list)
         for panel in self.panels:
             for hist in self.panels[panel].hists:
-                cols[panel].append(all_cols[counter % len(all_cols)])
+                colors[panel].append(all_colors[counter % len(all_colors)])
                 counter += 1
-        return cols
+        return colors
 
-    def _makePanel(self, data, panel, ax, col, **kwargs):
+    def _makePanel(self, data, panel, ax, colors, label_font_size=9, legend_font_size=7, ncols=1, **kwargs):
```

Comment:
As far as I can see, it doesn't look like these `kwargs` are passed anywhere, so perhaps they can be removed?